bigbluebutton-html5/imports/startup/server/index.js

```diff
@@ -12,7 +13,16 @@ import Redis from './redis';
 import setMinBrowserVersions from './minBrowserVersion';
 
 let guestWaitHtml = '';
-const AVAILABLE_LOCALES = fs.readdirSync('assets/app/locales');
+
+const meteorRoot = fs.realpathSync(`${process.cwd()}/../`);
+let applicationRoot = fs.realpathSync(`${meteorRoot}/../`);
+
+// if running on dev mode
+if (path.basename(fs.realpathSync(`${meteorRoot}/../../../`)) === '.meteor') {
```

Comment:
please remove this line and instead change applicationRoot to a const again and check `process.env.NODE_ENV || 'development'` for deciding on which of the two paths to assign to the constant.